assets/modal-form.js

```diff
@@ -50,6 +50,11 @@
             } else if (clientOptions.tabindex == false) {
                 $modalDialog.removeAttr('tabindex');
             }
+            if (clientOptions.backdrop) {
+                $modalDialog.attr('data-backdrop', clientOptions.backdrop);
+            } else if (clientOptions.backdrop == false) {
```

Comment:
clientOptions.backdrop === false 
Is strict comparison required?

Reply:
You are right, it's not required

Reply:
So, if (clientOptions.backdrop == false) - this comparison is also not required

Reply:
so I change the code and raise the pull request?